styleguide/screens/Patterns/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Dropdown, { HORIZONTAL_ATTACHMENTS } from '../../../../components/Dropdown/Dropdown';
+import Specimen from '../../../components/Specimen/Specimen';
+import { H, T, D, A, C, Bq, List, Placeholder } from '../../../components/Scaffold/Scaffold';
+
+import m from '../../../../globals/modifiers.css';
+import css from './Dropdown.css';
+
+// eslint-disable-next-line react/prop-types
+const Target = ({ onClick, placeholderText }) => (
+  <Placeholder onClick={ onClick }>{ placeholderText }</Placeholder>
+);
+
+export default class DropdownDocumentation extends Component {
+  state = {
+    showDropdown: false,
+  };
+
+  handleClick = () => {
+    this.setState(({ showDropdown }) => ({
+      showDropdown: !showDropdown,
+    }));
+  };
+
+  render() {
+    const { showDropdown } = this.state;
+
+    return (
+      <div>
+        <H level={ 1 }>Dropdown</H>
+        <T elm="p" className={ cx(m.mtr, m.largeI, m.demi) }>
+          Dropdowns are toggleable overlays. They’re typically used to add additional information or
+          functionality to a given element. Only trigger the display of a dropdown using
+          a <C>click</C> or <C>hover</C>.
```

Comment:
Click and hover don't need to be code snippets and I think we should drop "or hover" as it provides a suboptimal experience for touch device users